main.nf

```diff
@@ -1035,19 +1067,22 @@ process multiqc {
     tag "$prefix"
     publishDir "${params.outdir}/MultiQC", mode: 'copy'
 
+    when:
+    !params.skip_multiqc
```

Comment:
I think this `when` block can also be added to [workflow_summary_mqc](https://github.com/nf-core/rnaseq/blob/master/main.nf#L1016) process (?) Since that is also used only for `MultiQC`.

And add `.ifEmpty([])` to [this](https://github.com/nf-core/rnaseq/blob/master/main.nf#L1083) line and `!param.skip_multiqc` to [this](https://github.com/nf-core/rnaseq/blob/master/main.nf#L1133) `if` condition, just in case `MultiQC` was not run.